### DISCUSSION TITLE
Assistants API

### DISCUSSION THREAD

**Discussion by fgalastri** [microsoft/aisdkforsapabap#35] [General]
Good afternoon ! 

Any plans on releasing access to the assistants API ? 

Regards, 

Fabiano.

**Comment by gopalnair** [comment]
Hi @fgalastri ,
Of course! We will build the capability into the library in near future.

Things are changing so fast in the APIs that, the thought was to wait a bit for things to stabilize before building it into the sdk.

Best Regards, 
Gopal Nair
